aptos-move/aptos-intent/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "aptos-intent"
+description = "Generating Move Script from intent"
+version = "0.1.0"
+
+# Workspace inherited keys
+authors = { workspace = true }
+edition = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+publish = { workspace = true }
+repository = { workspace = true }
+rust-version = { workspace = true }
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+anyhow = { workspace = true }
+bcs = { workspace = true }
+bytes = { workspace = true }
+move-binary-format = { workspace = true }
+move-bytecode-verifier = { workspace = true }
+move-core-types = { workspace = true }
+serde = { workspace = true }
+serde_bytes = { workspace = true }
+serde_json = { workspace = true }
+hex = { workspace = true }
+getrandom = { version = "0.2", features = ["js"] }
+wasm-bindgen = "0.2"
+wasm-bindgen-futures = "0.4.42"
+reqwest-wasm = { version = "*", features = ["blocking"] }
+reqwest = { version = "*", features = ["blocking"] }
+
+[dev-dependencies]
+e2e-move-tests = { path = "../e2e-move-tests" }
+aptos-types = { workspace = true }
```

Comment:
I assume the funky symbol means there's no trailing `\n` on this file?  I  think you need one to be POSIX-compliant.